.gitignore

```diff
@@ -127,9 +127,6 @@ ENV/
 .idea/
 *.iml
 
-# Visual Studio Code
-.vscode/
```

Comment:
Had to stop ignoring `.vscode` to make some auto-save settings apply to everyone.

Reply:
I don’t see any changes in `.vscode` in this PR. Why is this change required?

Reply:
Great catch! There was another line in `.gitignore` making `.vscode/settings.json` uncommiteable. Just pushed the fix. Thank you @uranusjr!